Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcServer : IRpcServer
+    {
+        private readonly INetqRpcRabbitConnection<Func<Type,object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        private IDisposable _respondHandler;
+
+        public NetqRabbitRpcServer(INetqRpcRabbitConnection<Func<Type,object>> netqRpcRabbitConnection, ILoggerFactory loggerFactory)
+        {
+            _netqRpcRabbitConnection = netqRpcRabbitConnection;
+            _logger = loggerFactory.CreateLogger(nameof(NetqRabbitRpcServer));
+            _respondHandler = null;
```

Comment:
```suggestion
            _netqRpcRabbitConnection = netqRpcRabbitConnection;
            _respondHandler = null;
            _logger = loggerFactory.CreateLogger(nameof(NetqRabbitRpcServer));
```

Same as above , initialize the by the same order. Again i think that the logger looks more good in the end